packages/terra-table/src/Table.scss

```diff
@@ -154,12 +154,12 @@
 
     &.row:focus,
     &.row:hover {
-      background-color: var(--terra-table-row-hover-background-color, #ebf6fd) !important;
+      background-color: var(--terra-table-row-hover-background-color, lighten(#ebf6fd, 2%)) !important;
     }
 
     &.is-selected:focus,
     &.is-selected:hover {
-      background-color: var(--terra-table-row-selected-hover-background-color, darken(#ebf6fd, 5%)) !important;
+      background-color: var(--terra-table-row-selected-hover-background-color, darken(#ebf6fd, 7%)) !important;
```

Comment:
This isn't in the scope of this change, but are all these important tags needed anymore after uplifting to CSS modules?